controllers/controllers.js

```diff
@@ -0,0 +1,7 @@
+const {selectTopics} = require('../models/models');
+
+exports.getTopics = (req, res, next) => {
+    selectTopics().then((allTopics) => {
+        res.status(200).send({allTopics})
+    })
```

Comment:
.catch

Reply:
@k-huggs do i still need the .catch if my error won't be coming from model/controller? I thought this endpoint won't be triggered if the path is incorrect...

Reply:
true, however, I would always put one on just incase something unusual came back